### ISSUE TITLE
Typo in documents?

### ISSUE THREAD

**Issue by km19809** [lemunozm/message-io#129]
In [node.rs](https://github.com/lemunozm/message-io/blob/master/src/node.rs#L379):
```rs
let task = listener.for_each(move |event| match event {
     NodeEvent::Network(net_event) => { /* Your logic here */ },
     NodeEvent::Signal(_) => handler.stop(),
});
```
The example of `for_each_async` uses `for_each`.

**Comment by lemunozm** [comment]
Hi @km19809, your right! 

It's already fixed in master, although it will not appear in the documentation until the next minor release.

Thanks for notifying.

**Comment by km19809** [comment]
Thank you!

Closed with 5dc935eaaefbb10d43f45f55c7cc7f812243d75b
